tests/test_elk_reporter.py

```diff
@@ -404,3 +404,27 @@ def test_2():
 
     second_report = json.loads(requests_mock.request_history[1].text)
     assert "mark2" in second_report["markers"]
+
+
+def test_user_properties(testdir, requests_mock):
+    testdir.makepyfile(
+        """
+        import pytest
+
+        def test_1(record_property):
+            record_property("example_key", 1)
```

Comment:
`record_property` is a fixture ? cause I don't see it in the code

Reply:
looks like I'm not update with my pytest game :crying_cat_face: 
https://docs.pytest.org/en/stable/reference.html#pytest.junitxml.record_property